include/ibamr/IBFEMethod.h

```diff
@@ -636,8 +428,8 @@ class IBFEMethod : public IBStrategy
      * @note Since this class assumes that structures live on the finest grid
      * level the second argument is ignored.
      */
-    virtual void inactivateLagrangianStructure(int structure_number = 0,
-                                               int level_number = std::numeric_limits<int>::max()) override;
+    void inactivateLagrangianStructure(int structure_number = 0,
+                                       int level_number = std::numeric_limits<int>::max()) override;
```

Comment:
Can we move this and related functions into `FEMechanicsBase`? Since that class already stores data structures per part, it makes more sense that these should be there.

Reply:
I can move the data structure, but `inactivateLagrangianStructure` and related functions are in `IBStrategy`, so we cannot completely get rid of them in `IBFEMethod`.

Reply:
Oh duh. Lets definitely not move this into `FEMechanicsBase` - its too complicated to think about which parent class has the right virtual function definition.